apis/kubedb/v1alpha2/pgbouncer_helpers.go

```diff
@@ -298,3 +301,38 @@ func (p *PgBouncer) setConnectionPoolConfigDefaults() {
 		p.Spec.ConnectionPool.IgnoreStartupParameters = PgBouncerDefaultIgnoreStartupParameters
 	}
 }
+
+func (p *PgBouncer) SetSecurityContext() {
+	if p.Spec.PodTemplate.Spec.ContainerSecurityContext == nil {
+		p.Spec.PodTemplate.Spec.ContainerSecurityContext = &core.SecurityContext{
+			RunAsUser:  pointer.Int64P(70),
```

Comment:
We don't hardcode things from here. We should put those in the PgBouncerVersion and take from there. Like we do for Postgres

https://github.com/kubedb/installer/blob/master/catalog/raw/postgres/postgres-15.1-official.yaml#L18-L20